guides/rails_guides/markdown/renderer.rb

```diff
@@ -101,8 +101,7 @@ def github_file_url(file_path)
                    "#{root.sub("_", "")}/lib/#{file_path}"
                  else
                    file_path
-                 end
-
+          end
```

Comment:
This seems pretty terrible. Which rubocop setting thinks this is a good idea? Maybe we just need to change its mind.